dir.props

```diff
@@ -10,7 +10,7 @@
 
   <!-- Build Tools Versions -->
   <PropertyGroup>
-    <BuildToolsVersion>1.0.25-prerelease-00134</BuildToolsVersion>
+    <BuildToolsVersion>1.0.25-prerelease-00137</BuildToolsVersion>
```

Comment:
Verified 141 builds in TFS without issue. I'm also going to modify my PR to have an extra prior commit to the build script and build instructions with the version from CoreFx so that the diff can easily be seen.